app/main.py

```diff
@@ -1,8 +1,53 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class  # from 1 to 7
+        self.clean_mark = clean_mark  # from dirty - 1 to clean - 10
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center  # 1.0 to 10
+        self.clean_power = clean_power  # up to clean_mark
+        self.average_rating = average_rating  # from 1.0 to 5.0
+        self.count_of_ratings = count_of_ratings  # number of voting
+
+    """
+    returns income of CarWashStation for serving list of Car's
+    """
+    def serve_cars(self, list_cars: list[Car]) -> float:
+        return round(sum([self.wash_single_car(car) for car in list_cars]), 1)
```

Comment:
Docstring should be inside function. Fix everywhere.
```suggestion
    def serve_cars(self, list_cars: list[Car]) -> float:
        """
        returns income of CarWashStation for serving list of Car's
        """
        return round(sum([self.wash_single_car(car) for car in list_cars]), 1)
```